theme/management/commands/dump_user_quota.py

```diff
@@ -71,9 +72,11 @@ def handle(self, *args, **options):
             uz_bytes = None
             dz_bytes = None
             try:
-                uz_bytes, dz_bytes = get_quota_usage(user.username)
+                uz_bytes, dz_bytes = get_quota_usage(user.username, raise_on_error=False)
```

Comment:
Why make this change? With this change I assume there would be no exception `ValidationError` which we are trying to handle here.

Reply:
thanks